app/Http/Controllers/SportsController.php

```diff
@@ -18,15 +20,55 @@ public function create()
 
     public function store(Request $request)
     {
-        // Add your code here
+
+        $country = new Country;
+
+        foreach ($request->except('_token') as $key => $value) {
+
+            // if user not select country it will be default 0
+
+            $validator = Validator::make(
```

Comment:
@wahdam-amar interesting solution with multiple validators. Maybe possible to have it all in one form request class?